### PR TITLE
Add Sales Grid app.

### PR DESCRIPTION
**Pull request by skrustev**

### CODE REVIEW

**Comment by dkamburov** [comment]
1. There is an error in the console, even though the sample is running:
image
2. Clicking on Store dimension is really slow action, makes the grid not responsive for more than 10 seconds(or dragging the periods column). Is it related to the data being bigger than it is in the samples? And what can we do about it?
3. The grid is not taking the whole space of the document. Is this by design?
image

**Comment by skrustev** [comment]
@dkamburov 
1. This error is throwing only when running the sample locally using `npm start`. Running it as a client there are no such errors, so I am not sure what could it be yet.
2. The data used currently for the sample is 100 000 records yes. It takes a bit for it to be processed into dimensions through the pipes. Maybe a product feature would be to make the grid not lock the window and display a loading screen when doing such operations, but that is not present at the moment.
3. For now I've made it similar to the design size. When the whole app is created it can be tweaked to fit the parent container, whatever size it is.

File: projects/SalesGrid/src/index.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>SalesGridApp</title>
+  <base href="/">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link href="https://fonts.googleapis.com/css?family=Titillium+Web" rel="stylesheet">
```
Comment (mddragnev) [inline projects/SalesGrid/src/index.html:9]: don't need this

**Review by mddragnev — APPROVED** [review]

**Comment by yradoeva** [comment]
Here are my comments, related to design:

1. For the list in the side panel - I know it was discussed to be without paddings, but it doesn't look ok when you hover the items, so let's make the paddings 0px 8px
current:
Q Search

to be:
image

2. The flags should be with height 14px - 18.67px/14px (not 21.33/16px)
Germany

3. Title in the toolbar should be “Quick Action**s**”

4. Filter chip dropdown - use the same order of the items in both, let's order them alphabetically as in the second
Average


5. Export icon to be the custom one: indigo-file-download
Pasted Graphic 27

**Pull request merged**